.github/workflows/nix-dev-cache.yaml

```
@@ -24,6 +24,11 @@ jobs:
           - macOS-14
     steps:
     - uses: actions/checkout@v4
+    - name: set up nix mount points on Linux
+      if: runner.os == 'Linux'
+      run: |
+        sudo mkdir /nix /mnt/nix
+        sudo mount -B /mnt/nix /nix
```

```suggestion
        sudo mount --bind /mnt/nix /nix
```
I find it helpful to use expanded flags in scripts, more obvious what they’re doing.

---

Do you know if this definitely works on Github? I had tried `mount -o bind` initially and either it didn't work or I messed something else up.

---

I realized I can re-run and old commit to check.